### ISSUE TITLE
Workload information of "input_data.npy" under unet/

### ISSUE THREAD

**Issue by rtu715** [boringlee24/socc22-miso#1]
Hi, I am running some experiments with unet training, and I would like to know what the workloads in input_data.npy correspond to. Could you point me to a script that generated this data, or where I can match the data records with their respective MPS workloads?

**Comment by boringlee24** [comment]
Hi, the files in `mps/models/logs/mig` are the ground truth performance labels when running different jobs in MIG. They are generated by the `mps/run_emuerate.py` script. The `mps/collect_mps.py` script is used to generate the training data when running different combinations of jobs in MPS. To run collect_mps.py, you need to have MIG mode disabled and MPS mode enabled on the GPU, make sure the MPS-related directories are accessible, then run it as `python collect_mps.py --port 50051 --num_active 7`

**Comment by rtu715** [comment]
Thank you!
